translations/messages+intl-icu.en_GB.yaml

```diff
@@ -102,9 +102,12 @@ home.no_tickets: 'No tickets'
 home.title: 'Welcome to Bileto'
 home.your_organizations: '{count, plural, one {Your organization} other {Your organizations}}'
 home.your_tickets: 'Your tickets'
-hours_formatter.hours: '{hours}h'
-hours_formatter.hours_and_minutes: "{hours}h\_{minutes}m"
-hours_formatter.minutes: '{minutes}m'
+hours_formatter.hours: '{hours} h'
+hours_formatter.hours_and_minutes: "{hours} h\_{minutes} m"
+hours_formatter.minutes: '{minutes} m'
```

Comment:
Why did you add a space between the number and the unit?

Reply:
I thought the convention was to add a space between values ans the units sorry

Reply:
Your argument is good, don't worry :P That was my initial mistake!

Note that we should use non-breaking spaces so the number is not separated of its unit at the end of a line (see the figure 1 in this article https://intelligentediting.com/blog/non-breaking-spaces-how-to-check-something-you-cannot-even-see/ for instance). As we cannot distinguish a normal space from a non-breaking space, we can use the special character `\_` in YAML to represent a non-breaking space.